app/views/shared/_signup_form.html.erb

```diff
@@ -1,51 +1,47 @@
 <h3 class="signup-text"><%= title %></h3>
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {class: 'form-horizontal signup-form'}) do |f| %>
-
   <% if is_home == false %>
     <%= devise_error_messages! %>
   <% end %>
-
   <div class="form-group">
     <div class="col-xs-12">
-      <input class="form-control" placeholder="Full Name">
+      <input class="form-control" placeholder="Full Name" required="true">
     </div>
   </div>
-  
   <% if is_home %>
     <div class="form-group">
       <div class="col-xs-12 col-md-6">
-        <%= f.email_field :email, class: 'form-control', placeholder: 'Email' %>
+        <%= f.email_field :email, required: true, class: 'form-control', placeholder: 'Email' %>
       </div>
       <div class="col-xs-12 col-md-6">
-        <%= f.password_field :password, class: 'form-control', placeholder: 'Password', autocomplete: "off" %>
+        <%= f.password_field :password, required: true, class: 'form-control', placeholder: 'Password', autocomplete: "off" %>
       </div>
     </div>
   <% else %>
     <div class="form-group">
       <div class="col-xs-12">
-        <%= f.email_field :email, class: 'form-control', placeholder: 'Email' %>
+        <%= f.email_field :email, required: true, class: 'form-control', placeholder: 'Email' %>
       </div>
     </div>
     <div class="form-group">
       <div class="col-xs-12">
         <% if @minimum_password_length %>
           <em>(<%= @minimum_password_length %> characters minimum)</em>
         <% end %>
-        <%= f.password_field :password, class: 'form-control', placeholder: 'Password', autocomplete: "off" %>
+        <%= f.password_field :password, required: true, class: 'form-control', placeholder: 'Password', autocomplete: "off" %>
       </div>
     </div>
     <div class="form-group">
       <div class="col-xs-12">
-        <%= f.password_field :password, class: 'form-control', placeholder: 'Password Confirmation', autocomplete: "off" %>
+        <%= f.password_field :password, required: true, class: 'form-control', placeholder: 'Password Confirmation', autocomplete: "off" %>
       </div>
     </div>
   <% end %>
-
   <div class="form-group">
     <div class="col-xs-12">
       <div class="checkbox">
         <label>
-          <input type="checkbox"> I agree to the <%= link_to 'Erudition Terms', privacy_path, class: 'privacy-link' %>
+          <input type="checkbox" required="true"> I agree to the <%= link_to 'Erudition Terms', privacy_path, class: 'privacy-link' %>
```

Comment:
Lets not require this until we actually have some TOS. in fact, we should not even display it until we have TOS